server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/java/org/apache/james/jmap/draft/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -275,6 +275,38 @@ public void userShouldBeSubscribedOnCreatedMailboxWhenCreateMailbox() throws Exc
         assertThat(mailboxProbe.listSubscriptions(username)).contains("foo");
     }
 
+    @Category(BasicFeature.class)
+    @Test
+    public void userShouldGetAnErrorWhenNotBeAbleToCreateACaseVariationOfINBOX() throws Exception {
+        String requestBody =
+            "[" +
+                "  [ \"setMailboxes\"," +
+                "    {" +
+                "      \"create\": {" +
+                "        \"create-id01\" : {" +
+                "          \"name\" : \"iNbOx\"" +
+                "        }" +
+                "      }" +
+                "    }," +
+                "    \"#0\"" +
+                "  ]" +
+                "]";
+
+        given()
+            .header("Authorization", accessToken.serialize())
+            .body(requestBody)
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .body(NAME, equalTo("mailboxesSet"))
+            .body(ARGUMENTS + ".notCreated", hasKey("create-id01"))
+            .body(ARGUMENTS + ".notCreated.create-id01.type", equalTo("invalidArguments"))
+            .body(ARGUMENTS + ".notCreated.create-id01.description", equalTo("The mailbox 'create-id01' already exists."));
+
+        assertThat(mailboxProbe.listSubscriptions(username)).doesNotContain("iNbOx");
```

Comment:
listSubscriptions is not sufficient, it lists... the subscriptions, which can not include all the mailboxes...